### PR TITLE
Update video to have list of genres

### PR DESCRIPTION
**Pull request by ChuckBTaylor**

### CODE REVIEW

File: src/components/atoms/videoInfo/VideoInfo.tsx [hunk 1]
```diff
@@ -1,15 +1,16 @@
 import { Typography } from "@mui/material";
+import { ReactNode } from "react";
 
 type VideoInfoProps = {
   label?: string;
-  text: string;
+  infoValue: ReactNode
```
Comment (ChuckBTaylor) [inline src/components/atoms/videoInfo/VideoInfo.tsx:6]: @mwilkins74 , this should make it easy for you to pass in `<Link>` components.

File: src/assets/videos/videos.ts [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
 
-
 const videos: IVideoCard[] = [
   {
     url: "",
     title: "",
     clientName: "",
-    genre: "animation",
+    genres: ["animation"],
     roles: ["camera_operator"],
```
Comment (kmdunn5) [inline src/assets/videos/videos.ts:9]: Is entering things here as snake case going to be the easiest thing for Kurt? It'll help for type checking and validation, but just trying to think forward for our non-programmer friend who will have to remember all of this in 10 years.

**Review by kmdunn5 — APPROVED** [review]

File: src/assets/videos/videos.ts (hunk 2 above)
Comment (ChuckBTaylor) [reply src/assets/videos/videos.ts:9]: Good Question.  It.may not be the easiest to remember for him. We could look into changing that.  Good news is that it is type checked, so he'll know immediately if it's wrong. Same is true if it's not snake_case, though. So I'm indifferent.

**Comment by mwilkins74** [comment]
Looks great! 
I'll merge this in and then start re-building/passing in the VideoInfo.tsx to pass in Links.

**Pull request merged**